tools/fiddler/OneDSInspector/packages.config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Bond.Compiler" version="8.1.0" targetFramework="net461" />
-  <package id="Bond.Core.CSharp" version="8.1.0" targetFramework="net461" />
-  <package id="Bond.Runtime.CSharp" version="8.1.0" targetFramework="net461" />
-  <package id="Newtonsoft.Json" version="12.0.2" targetFramework="net461" />
+  <package id="Bond.Compiler" version="9.0.4" targetFramework="net461" />
+  <package id="Bond.Core.CSharp" version="9.0.3" targetFramework="net461" />
+  <package id="Bond.Runtime.CSharp" version="9.0.3" targetFramework="net461" />
+  <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net461" />
```

Comment:
was this intended to downgrade Json version ?

Reply:
Yes, I mentioned it in the PR notes: downgrade Newtonsoft to version employed by Fiddler 5.x to simplify installation. Currently Fiddler 5.x is shipping with that version. I made sure that we compile our Fiddler 5.x inspector against the same version that is installed with Fiddler 5 (they are not using latest).